templates/minimal/runtime/Cargo.toml

```diff
@@ -35,8 +34,9 @@ polkadot-sdk = { optional = true, workspace = true, features = [
 default = ["std"]
 std = [
 	"codec/std",
-	"pallet-minimal-template/std",
-	"polkadot-sdk/std",
 	"scale-info/std",
 	"serde_json/std",
+
+	"pallet-minimal-template/std",
+	"polkadot-sdk/std",
```

Comment:
personally I find keeping it in one block better.

Reply:
then it can also be sorted by toml linters

Reply:
reverted in https://github.com/paritytech/polkadot-sdk/pull/5995/commits/bd17cf205f6ddc934436eb2a3d68d63fb7e86b02